### ISSUE TITLE
How to use classifier..

### ISSUE THREAD

**Issue by edwardcho** [hyperconnect/LADE#5]
Hello Sir,

I checked DotProductClassifier.py and CausualNormClassifier.py in "models" folder of your source code.

If I use in other networks (ex: Inception_v1),
for LADE or Balanced softmax or Causal Norm, 
How to use this function in other networks?? 
(Should I do create new network using feature-extraction layers and your classifiers? )
(at Training and at Inference)

Thanks,
Edward Cho

**Comment by wade3han** [comment]
You can just replace the loss function and the training code for LADE and Balanced Softmax. You don't need to manipulate the model code.

**Comment by edwardcho** [comment]
Hello Sir,

Honesty I couldn't solve my issues.

I can understand your comment for LADE and balanced-softmax.
(Just replacing loss function, I can try it. I can understand)
Thanks...

But, still I didn't solve Causal-Norm in your code.
Should I do create new network using feature-extraction layers and your classifier for other networks ??

Thanks,
Edward Cho.
